pr2eus/robot-interface.l

```diff
@@ -1556,7 +1555,7 @@ send-action [ send message to action server, it means robot will move ]"
          (send msg :points (list pt1 pt2)))
        ))
      (send goal :goal :trajectory msg)
-     (when send-action and move-base-trajectory-action
```

Comment:
@k-okada Because of luck of test case on `go-pos-unsafe`?
I think it is used only for sending action to `pr2_base_trajectory_action` controller.